app/src/main/java/com/blackboxteam/app/blackbox/LoginScreenActivity.java

```diff
@@ -1,13 +1,79 @@
 package com.blackboxteam.app.blackbox;
 
-import android.support.v7.app.AppCompatActivity;
+import android.content.Intent;
 import android.os.Bundle;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.blackboxteam.app.blackbox.util.Validation;
+
+import static com.blackboxteam.app.blackbox.util.Validation.validateEmail;
+import static com.blackboxteam.app.blackbox.util.Validation.validateMinLength;
+
+public class LoginScreenActivity extends AppCompatActivity implements View.OnClickListener {
 
-public class LoginScreenActivity extends AppCompatActivity {
+    Button loginBtn;
+    EditText etEmail, etPassword;
+    TextInputLayout tiEmail, tiPassword;
+    Validation validate;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login_screen);
+
+        loginBtn = (Button) findViewById(R.id.btn_login);
+        etEmail = (EditText) findViewById(R.id.et_email);
+        etPassword = (EditText) findViewById(R.id.et_password);
+        tiEmail = (TextInputLayout) findViewById(R.id.ti_email);
+        tiPassword = (TextInputLayout) findViewById(R.id.ti_password);
+        loginBtn.setOnClickListener(this);
```

Comment:
I believe we have data  binding enabled for this project. Let's use it adequately and refrain from using findViewById;